src/appleseed/renderer/modeling/scene/assembly.cpp

```diff
@@ -56,6 +56,8 @@ namespace renderer
 // Assembly class implementation.
 //
 
+const char* Model = "assembly";
```

Comment:
Or even simpler, we leave everything as is for now, i.e. we keep `Assembly` (of model `generic_assembly`).